tex/chapters/03-Theory.tex

```diff
@@ -3,88 +3,152 @@
 \chapter{Theory}\label{cha:theory}
 
 Many definitions will be now be presented, with the overall aim being to have a rigorous definition for a fingerprint.
-Definitions for the Dual, Joss-Ann and Fingerprint were first presented by Ashlock in \cite{Ashlock2004} as mentioned in Section \ref{sec:fingerprinting}.
-Then a definition of Finite State Machines will be given, and an explanation of how they relate to fingerprinting.
-Finally, an example of how to construct a fingerprint will be shown in Section \ref{sec:analytical-fingerprints}.
+A definition of Finite State Machines will be given, and an explanation of how they relate to fingerprinting.
+Then definitions for the Dual, Joss-Ann and Fingerprint as presented by Ashlock in \cite{Ashlock2004}.
+It will then be shown that any strategy can be represented as a Finite State Machine, which allows the work of Ashlock to be extended to include any strategy to be used as a probe.
+Finally, an example of how to construct an analytical fingerprint will be shown in Section \ref{sec:analytical-fingerprints}.
+In essence this chapter presents a detailed review of the work of \cite{Ashlock2008, Ashlock2010, Ashlock2004,  Ashlock2005, Ashlock2009, Ashlock2006,}.
+
+
+
+
+\section{Finite State Machines}\label{sec:fsm}
+
+A formal definition of a Finite State Machine is given by Definition \ref{def:fsm} but first we will outline some motivating key characteristics of a system that can be modelled with a FSM:
+
+\begin{itemize}
+ \item The system must be describable by a finite set of states.
+ \item The system must have a finite set of inputs that can trigger transitions between states.
+ \item The behaviour of the system at a given point in time depends upon the current state and the input that occurs at that time.
+ \item For each state the system may be in, behaviour is defined for each possible input.
+ \item The system has a particular initial state.
+\end{itemize}
+
+We can make the above bullet points rigorous for deterministic cases with the following definition:
```

Comment:
Add a couple of references. Without them, this looks like you're defining the FSMs.